docs/source/manual/upgrade-notes/upgrade-notes-2_1_x.rst

```diff
@@ -146,3 +146,27 @@ However, if you are experiencing issues with the client modifications, you can e
         .build();
 
 The property ``HttpUrlConnectorProvider.SET_METHOD_WORKAROUND`` is optional and should be avoided on Java versions after Java 15.
+
+Allow parameter conversion exceptions to be mapped to responses by exception mappers
+====================================================================================
+
+Some parameter validation logic has been fixed to allow for custom remapping of error messages. As a result, the API responses may be a little different than before in certain scenarios.
+
+The responses may be different than before this change if the default exception mappers are disabled, or custom exception mappers are registered:
+
+* If the default exception mappers provided by Dropwizard are disabled (``registerDefaultExceptionMappers`` option is off) and no other exception mappers are configured, then the response will have the same status code as before this change, but with no entity or media type.
+* If an exception mapper is registered which takes precedence over ``LoggingExceptionMapper`` for mapping ``WebApplicationException`` instances, then the response will be generated by that exception mapper. The ``WebApplicationException`` instance received by the exception mapper will have the same status code and message as the response had before this change.
+
+In addition, the ``AbstractParam.mediaType()`` method has been deprecated, and the responses may not use the media type returned by that method.
+
+There are two types of parameters that may result in different error responses in the above situations if there is a validation error:
+
+* Enum parameters
+* Parameters that are any subclass of ``AbstractParam`` (Dropwizard provides the subclasses ``InstantParam``, ``InstantSecondParam``, ``LocalDateParam``, ``LocalDateTimeParam``, ``LocalTimeParam``, ``OffsetDateTimeParam``, ``YearMonthParam``, ``YearParam``, ``ZonedDateTimeParam``, ``ZoneIdParam``, ``BooleanParam``, ``DateTimeParam``, ``DurationParam``, ``InstantParam``, ``IntParam``, ``LocalDateParam``, ``LongParam``, ``NonEmptyStringParam``, ``SizeParam``, and ``UUIDParam``)
+
+If this code change causes an undesired change in the error responses:
+
+* If the error responses are not being mapped to a response by any exception mapper, then either register the default exception mappers (by enabling the ``registerDefaultExceptionMappers`` option) or register Dropwizard's ``ThrowableMapper`` to restore the previous behavior, or register your own custom exception mapper that maps ``WebApplicationException`` instances.
```

Comment:
I think you meant `LoggingExceptionMapper` here.
```suggestion
* If the error responses are not being mapped to a response by any exception mapper, then either register the default exception mappers (by enabling the ``registerDefaultExceptionMappers`` option) or register Dropwizard's ``LoggingExceptionMapper`` to restore the previous behavior, or register your own custom exception mapper that maps ``WebApplicationException`` instances.
```

Reply:
@pstackle Thanks for finding my typo, I've squashed the release note changes into one commit and rebased.